_test/test_mpi/test_ParpoolMPI_Framework.m

```diff
@@ -97,34 +92,30 @@ function test_finish_tasks_reduce_messages(obj,varargin)
             [ok,err,mess] = serverfbMPI.receive_message(1,'completed');
             assertEqual(ok,MESS_CODES.ok,err);
             assertEqual(mess.mess_name,'completed');
-            
+
         end
         %
         function test_probe_all_receive_all(obj,varargin)
             % common code -------------------------------------------------
             if obj.ignore_test
-                return;
+                skipTest();
```

Comment:
If this code is now registering something - a skip - rather than silently existing, worth providing a reason argument so people seeing the skip can know to ignore it

Reply:
It defaults to "skipped test". The warning is provided higher up when ignore test is set.